backfill_corrections/Makefile

```
@@ -33,19 +54,41 @@ run-local:
 	[ "$$?" -eq 1 ]
 
 run:
-	# Assumes receiving is output dir
 	docker run --rm --pull=always \
-	    -v "${PWD}"/logs:/backfill_corrections/logs \
-	    -v "${PWD}"/receiving:/backfill_corrections/receiving \
-	    -v "${PWD}"/input:/backfill_corrections/input \
-	    -v "${PWD}"/cache:/backfill_corrections/cache \
+	    -v "${PWD}/${LOG_DIR}:/backfill_corrections/${LOG_DIR}" \
+	    -v "`realpath $(USR_EXPORT_DIR)`:/backfill_corrections/${EXPORT_DIR}" \
+	    -v "`realpath $(USR_INPUT_DIR)`:/backfill_corrections/${INPUT_DIR}" \
+	    -v "`realpath $(USR_CACHE_DIR)`:/backfill_corrections/${CACHE_DIR}" \
 	    -v "${PWD}"/params.json:/backfill_corrections/params.json \
 		-i "${DOCKER_IMAGE}:${DOCKER_TAG}" \
 		/bin/bash -c "make run-local OPTIONS=\"${OPTIONS}\""
 
-deploy: run
-	# Assumes receiving is output dir
-	aws s3 cp receiving/ $(S3_BUCKET)/ --recursive --exclude "*" --include "*csv.gz" --acl public-read
+publish:
+ 	aws configure set aws_access_key_id AWS_KEY_ID
+ 	aws configure set aws_secret_access_key AWS_SECRET_KEY
```

did you mean these to be variables?
```suggestion
 	aws configure set aws_access_key_id $(AWS_KEY_ID)
 	aws configure set aws_secret_access_key $(AWS_SECRET_KEY)
```